### PR TITLE
ET-4032 moving slack alarms

### PR DESCRIPTION
**Pull request by jorge-cr-13**
## What 🕵️ 🔍

- Moving slack notifications to monitoring
- SOPS file with slack url
- Moving alarms for b2b & embedding prod to monitoring
- Add monitoring budget

Todo:

- update version after https://github.com/xaynetwork/xayn_terraform_modules/pull/40 has been merged
- apply changed on waf

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

File: monitoring/global/iam/roles/create_alarms/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  config_path = "../../policies/create_alarms"
+}
+
+locals {
+  account_role = {
+    917039226361 : "AWSReservedSSO_AdministratorAccess_d5667b4ba53e9276"
+    165060789769 : "AWSReservedSSO_AdministratorAccess_4387aec2e456a72d"
+    987436120296 : "AWSReservedSSO_AdministratorAccess_d0db2c3624489273"
+    765086940592 : "AWSReservedSSO_AdministratorAccess_c5576577adce5e76"
+    985612346564 : "AWSReservedSSO_AdministratorAccess_25de7fa586ff018d"
```
Comment (Robert-Steiner) [inline monitoring/global/iam/roles/create_alarms/terragrunt.hcl:19]:
```suggestion
    987436120296 : "AWSReservedSSO_AdministratorAccess_d0db2c3624489273"
    985612346564 : "AWSReservedSSO_AdministratorAccess_25de7fa586ff018d"
```

should we remove the once that are currently don't set any alarms?

File: embedding_service/infra/dev/env.hcl [hunk 2]
```diff
@@ -2,7 +2,7 @@ locals {
   name           = "dev"
   aws_account_id = 765086940592
   profile        = "AdministratorAccess-765086940592"
-  version        = "v0.28.0"
+  version        = "dbacddae45b470a42a83d490375ab8d197836477"
```
Comment (joecks) [inline embedding_service/infra/dev/env.hcl:5]: should be updated

File: monitoring/global/iam/roles/create_alarms/terragrunt.hcl (hunk 1 above)
Comment (joecks) [reply monitoring/global/iam/roles/create_alarms/terragrunt.hcl:19]: lass drin

File: monitoring/account.hcl [hunk 3]
```diff
@@ -0,0 +1,10 @@
+locals {
+  name           = "monitor"
+  aws_account_id = 365373133059
+  profile        = "AdministratorAccess-365373133059"
+  source_url     = "github.com/xaynetwork/xayn_terraform_modules"
+  version        = "40b062c7feca679fcc27efbd6e2f2893c759474d"
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply monitoring/account.hcl:6]: done!

File: embedding_service/infra/prod/env.hcl [hunk 4]
```diff
@@ -2,7 +2,7 @@ locals {
   name           = "prod"
   aws_account_id = 985612346564
   profile        = "AdministratorAccess-985612346564"
-  version        = "v0.28.0"
+  version        = "dbacddae45b470a42a83d490375ab8d197836477"
```
[the remark this answers is not in the document]
Comment (Robert-Steiner) [reply embedding_service/infra/prod/env.hcl]: done!

**Review by joecks — APPROVED** [review]

**Pull request merged**